AutocompleteClient/FW/Logic/Request/CIORecommendationsQuery.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CIORecommendationsQuery.swift
+//  AutocompleteClient
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+public struct CIORecommendationsQuery: CIORequestData {
+    public let maximumNumberOfResult: Int
+    public let pod: String
+    public let itemID: String
```

Comment:
Let's turn `itemID` into an array.  As an example, the recommendations for a shopping cart full of items could have ID's for all of the items in the cart in the array.